pr-dhl-woocommerce/assets/js/pr-dhl.js

```diff
@@ -506,6 +515,33 @@ jQuery( function( $ ) {
 
 			return false;
 		},
+
+		enable_disable_PDDP: function () {
+			var selected_product = $( '#pr_dhl_product' ).val();
+
+			if( selected_product === 'V53WPAK' ) {
```

Comment:
```suggestion
			if ( 'V53WPAK' === selected_product ) {
```
let's standardize the condition using Yoda notation.